set.mm

```diff
@@ -467694,6 +467784,45 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
       CADGSNHOSRDEFSNPQ $.
   $}
 
+  ${
+    $d X x $.  $d Y x $.
+    lbslsat.v $e |- V = ( Base ` W ) $.
+    lbslsat.n $e |- N = ( LSpan ` W ) $.
+    lbslsat.z $e |- .0. = ( 0g ` W ) $.
+    lbslsat.y $e |- Y = ( W |`s ( N ` { X } ) ) $.
+    $( The basis of an atom.  (Contributed by Thierry Arnoux, 20-May-2023.) $)
```

Comment:
I've never seen the word "atom" being used in this setting in the literature. Does it just mean "a line"? I don't feel strongly about this, but maybe you could write more explicitly like "A nonzero vector ` X ` is a basis of a line spanned by ` X `."

Reply:
Yes, I was referring to `LSAtom` as defined in Norm's Mathbox, without wanting to pull that definition out. That's exactly what theorem ~[lsatlspsn2](https://us.metamath.org/mpeuni/lsatlspsn2.html) is. 
"Line" and your proposal is probably better, but I could refer to Norm's definition anyway.